src/SprykerEco/Zed/Braintree/Persistence/Mapper/BraintreePersistenceMapperInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Braintree\Persistence\Mapper;
+
+use Generated\Shared\Transfer\PaymentBraintreeTransactionRequestLogTransfer;
+use Generated\Shared\Transfer\PaymentBraintreeTransactionStatusLogTransfer;
+use Generated\Shared\Transfer\PaymentBraintreeTransfer;
+use Orm\Zed\Braintree\Persistence\SpyPaymentBraintree;
+use Orm\Zed\Braintree\Persistence\SpyPaymentBraintreeTransactionRequestLog;
+use Orm\Zed\Braintree\Persistence\SpyPaymentBraintreeTransactionStatusLog;
+
+interface BraintreePersistenceMapperInterface
```

Comment:
I guess you don't need persistence layer mapper interface

Reply:
I don't see that rule in Spryker wiki so I prefer to add it